Controller/CategoryAdminController.php

```diff
@@ -40,8 +40,13 @@ public function listAction(Request $request = null)
         }
 
         $datagrid = $this->admin->getDatagrid();
+        $datagridValues = $datagrid->getValues();
 
-        if ($this->admin->getPersistentParameter('context')) {
+        $datagridContextIsNotSet = array_key_exists('context', $datagridValues) ?
```

Comment:
Avoid negative boolean variables please, it hurts the brain : `$datagridContextIsSet = …`